api/actors.yml

```diff
@@ -0,0 +1,149 @@
+models:
+  Actor:
+    type: object
+    properties:
+      kind:
+        type: string
+        readOnly: true
+        enum: [actor]
+      id: 
```

Comment:
I think this is intended to be its address, and we should use that name. There are multiple schemas for addresses, one of them confusing called "ID address".